### DISCUSSION TITLE
A graph only contains one node

### DISCUSSION THREAD

**Discussion by nuaawh** [pyg-team/pytorch_geometric#3778] [Q&A]
If a graph only contains one node (i.e., no edges, edges=[]), can the GCNConv layer be applied to such a problem ? 
If so, will the GCNConv layer degenerate into the fully connected layer ?

**Comment by rusty1s** [comment]
Yes, for empty `edge_index`, `GCNConv` is indeed equivalent to a single `torch.nn.Linear` layer.
